src/05-led-roulette/the-challenge.md

```diff
@@ -21,57 +21,77 @@ single period. This pattern will repeat itself every 800 ms. The Y axis labels e
 cardinal point: North, East, etc. As part of the challenge you'll have to figure out how each
 element in the `Leds` array maps to these cardinal points (hint: `cargo doc --open` `;-)`).
 
-Before you attempt this challenge, let me give you one last tip. Our GDB sessions always involve
+Before you attempt this challenge, let me give you one additonal tip. Our GDB sessions always involve
 entering the same commands at the beginning. We can use a `.gdb` file to execute some commands
 right after GDB is started. This way you can save yourself the effort of having to enter them
 manually on each GDB session.
 
-Place this `openocd.gdb` file in the root of the Cargo project, right next to the `Cargo.toml`:
+Using an editor create `openocd.gdb` in the root of the Cargo project, right next to the `Cargo.toml`:
 
 ``` console
-$ cat openocd.gdb
+vi openocd.gdb
```

Comment:
```suggestion
nano openocd.gdb
```
Not to start an editor flamewar here but can we suggest using `nano` instead?
`nano` displays underneath what the commands are. If you do not know how to use `vi`, you are kind of trapped.
`nano` should also be available everywhere.

Reply:
I agree that `nano` is a better choice in such circumstances for the mentioned reasons.

Reply:
Done